.github/workflows/pr.yaml

```diff
@@ -53,21 +52,51 @@ jobs:
         run: |
           sudo pip3 install dotrun requests==2.31.0 # requests version is pinned to avoid breaking changes, can be removed once issue is resolved: https://github.com/docker/docker-py/issues/3256
           chmod -R 777 .
+          echo "FLASK_DISABLE_AUTH_FOR_TESTS=true" >> .env
```

Comment:
@immortalcodes 
Is there a reason this is set here and commented out in the .env file 
https://github.com/canonical/assets.ubuntu.com/pull/309/files#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055dR30